kyuubi-assembly/pom.xml

```diff
@@ -94,6 +94,18 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>jline</groupId>
+            <artifactId>jline</artifactId>
+            <version>${jline.version}</version>
+            <exclusions>
+                <exclusion>
+                    <artifactId>junit</artifactId>
+                    <groupId>junit</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
Sorry, I just notice you pin `<version>` and `<exclusions>` directly in child pom `<dependencies>` block, could you please move them to root pom `<dependencyManagement>` block? @murong00

Reply:
@pan3793 Sure, refined in #1416, please help to take a look, thanks.